### DISCUSSION TITLE
Pagination in URL not working with lazy=lazy

### DISCUSSION THREAD

**Discussion by gofish543** [livewire/livewire#6527] [7. Bugs (archived)]
### Livewire version

3.0.0

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

Create a Paginated component
```php
<?php

namespace App\Livewire;

use App\Models\Rider;
use Illuminate\Contracts\View\View;
use Illuminate\Database\Eloquent\Builder;
use Livewire\Attributes\Computed;
use Livewire\Attributes\Locked;
use Livewire\Component;
use Livewire\WithPagination;

class RidersDataTable extends Component
{
    use WithPagination;

    /**
     * @lifecycle
     */
    public function mount(): void
    {
        //...
    }

    /**
     * @lifecycle (lazy)
     */
    public function placeholder(): View
    {
        return view('livewire.utilities.loading');
    }

    /**
     * @lifecycle
     */
    public function render(): View
    {
        return view('livewire.riders-data-table', [
            'riders' => Rider::query()
                ->paginate(
                    perPage: 10,
                    pageName: "riders_page"
                ),
        ]);
    }
}
```

Mark the component as lazy in the view
```php
 <livewire:riders-data-table/>
```

Observe the url does not update when a page is changed

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@gofish543 thanks for reporting! Any chance you could submit a failing test PR so we can investigate? Thanks!

**Comment by gofish543** [comment]
Undergoing a major application release last, this, and next week. I will circle back to this

**Comment by PhiloNL** [comment]
Thanks for reporting. I've added failing test https://github.com/livewire/livewire/pull/6760.

**Comment by adesugbaa** [comment]
Hi. Any progress on this - I spent hours troubleshooting. If I remove the #[Lazy] attribute it works as expected, but I prefer the lazy mode where the site feels more responsive.

**Comment by milosb793** [comment]
+1 Facing the same issue here

**Comment by onemoreahmad** [comment]
same issue

**Comment by richmondb** [comment]
experiencing the same issue on latest version of livewire

**Comment by oeholmen** [comment]
Any progress on this?

**Comment by onemoreahmad** [comment]
Lazy with wire:navgiate and pagination still doesn't work.

**Comment by TheSycoon** [comment]
i have facing same issue

**Comment by onemoreahmad** [comment]
I fixed it by hardcoded the loading icon directly in the placeholder method without using a view()

```
function placeholder()
    {
        return '<div> 
            <svg xmlns="http://www.w3.org/2000/svg" class="icon icon-tabler icon-tabler-loader animate-spin h-5 w-5"
                viewBox="0 0 24 24"
                stroke-width="1.5"
                stroke="currentColor"
                fill="none"
                stroke-linecap="round"
                stroke-linejoin="round"
            >
                <path stroke="none" d="M0 0h24v24H0z" fill="none" />
                <line x1="12" y1="6" x2="12" y2="3" />
                <line x1="16.25" y1="7.75" x2="18.4" y2="5.6" />
                <line x1="18" y1="12" x2="21" y2="12" />
                <line x1="16.25" y1="16.25" x2="18.4" y2="18.4" />
                <line x1="12" y1="18" x2="12" y2="21" />
                <line x1="7.75" y1="16.25" x2="5.6" y2="18.4" />
                <line x1="6" y1="12" x2="3" y2="12" />
                <line x1="7.75" y1="7.75" x2="5.6" y2="5.6" />
            </svg> 
    </div>';
}
```

**Comment by joshhanley** [comment]
> Thanks for reporting. I've added failing test #6760.



For those still experiencing this issue, does the above failing test https://github.com/livewire/livewire/discussions/6527#discussioncomment-6967301 submitted by @PhiloNL cover your situation?

If not could somone please submit a failing test PR? Follow how he did it in the above PR.

Thanks!
